functions/src/plugins/size.ts

```diff
@@ -72,43 +72,49 @@ export class SizeTask extends Task {
 
     const targetBranchArtifacts = await this.getTargetBranchArtifacts(pr);
 
-    const largestIncrease = await this.findLagestIncrease(targetBranchArtifacts, newArtifacts);
+    const largestIncrease = await this.findLargestIncrease(targetBranchArtifacts, newArtifacts);
 
     const failure = this.isFailure(config, largestIncrease.increase);
 
     if(failure) {
       const desc = `${largestIncrease.artifact.fullPath} increased by ${largestIncrease.increase} bytes`; // TODO pretty up bytes 
-      await this.setStatus(STATUS_STATE.Failure, desc, context, config);
+      await this.setStatus(STATUS_STATE.Failure, desc, config.status.context, context);
+      return ;
```

Comment:
I meant `return this.setStatus(STATUS_STATE.Failure, desc, config.status.context, context);` ;-)
same for the ones below